pages/dug_10/x-dasharo-dbx-uc-ci.md

```diff
@@ -0,0 +1,196 @@
+---
+theme: ../../slidev-template/theme
+layout: cover
+background: /intro.png
+class: text-center
+
+---
+
+## Enhancements in Dasharo CI: Automatic DBX and microcode refresh
+
+---
+
+## Agenda
+
+- Introduction
+- DBX and microcode
+- Automations
+- Closing thoughts
+
+---
+
+## Introduction
+
+- Michał Kopeć
+- Firmware Engineer working primarily with coreboot and EDK2 but also Heads and
+  Linux
+- Have been at 3mdeb for 4 years now
+- Free and open source software enthusiast
+- ThinkPad collector
+
+---
+
+## Key Security Components & Trust
+
+* **Microcode Updates:**
+  - Processor-level fixes/mitigations (e.g., Spectre, Meltdown).
+  - Applied very early by coreboot.
```

Comment:
This part could be covered in separate slides. How is ucode loaded?

https://igor-blue.github.io/2021/02/04/secure-boot.html#early-power-on

Reply:
Added a separate slide in https://github.com/Dasharo/presentations/pull/94/commits/a5f7ab7bf2fa41687adcae49280b18271153d330